src/fmllauncher/java/net/minecraftforge/fml/loading/moddiscovery/ModInfo.java

```diff
@@ -243,6 +245,11 @@ public IModInfo getOwner()
         {
             return owner;
         }
+
+        public String getDisplayURL()
+        { 
+            return displayURL;
+        }
```

Comment:
This should not be like this... This should be implemented from the interface, so this class doesn't need to be made public and you don't need to do those ugly instanceof checks.

Reply:
The instanceof checks aren't needed anyway, FML already hard assumes the implementation classes.

Reply:
I've now made a pull request for the interface. See MinecraftForge/ForgeSPI#6